bin/compile

```diff
@@ -17,30 +27,12 @@ function indent() {
 
 echo '-----> Installing binary dependencies for ActiveStorage Preview'
 
-SFFMPEG_VERSION="4.2.3"
-
-# Store which ffmpeg version was installed in the cache to bust the cache if it changes
-if [ -f $CACHE_DIR/.apt/SFFMPEG ]; then
```

Comment:
Should we keep some logic here that if `$CACHE_DIR/.apt/SFFMPEG` is there, we remove it? To purge old caches?